server/src/misc.go

```diff
@@ -306,16 +306,13 @@ func toSnakeCase(str string) string {
 	return strings.ToLower(snake)
 }
 
-func fixTableName(name string) string {
+func truncateTrimCheckEmpty(name string) string {
 	// Truncate long table names
 	// (we do this first to prevent wasting CPU cycles on validating extremely long table names)
 	if len(name) > MaxGameNameLength {
 		name = name[0 : MaxGameNameLength-1]
 	}
 
-	// Remove any non-printable characters, if any
-	name = removeNonPrintableCharacters(name)
```

Comment:
why

Reply:
removeNonPrintableCharacters was obsolete; valid table/seed name is handled by regex